### PR TITLE
saner punctuation for parenthesized full sentences

### PR DESCRIPTION
**Pull request by das-g**
When a full (and without the parenthesized part already complete) sentence is followed by a parenthesized full complete sentence, then end the former with a period before the parenthesis and end the latter with a period within the parenthesis (before the closing brace). Also, start
the parenthesized sentence with a capital letter.

Same for stand-alone parenthesized sentences.

### CODE REVIEW

File: en/deploy/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ $ git config --global user.name "Your Name"
 $ git config --global user.email you@example.com
 ```
 
-Initializing the git repository is something we need to do only once per project (and you won't have to re-enter the username and email ever again).
+Initializing the git repository is something we need to do only once per project. (And you won't have to re-enter the username and email ever again.)
```
Comment (ekohl) [inline en/deploy/README.md]: Starting a sentence with `and` doesn't sound right. In fact, the whole sentence could use some improvement to indicate it's global but a distinct separate step from `init`. How about `The config is done only once on your computer. Even when you start another git project on the same computer, you won't have to re-enter the username and email again.`

File: en/chromebook_setup/instructions.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ source myvenv/bin/activate
 pip install django~={{ book.django_version }}
 ```
 
-(note that on the last line we use a tilde followed by an equal sign: ~=).
+(Note that on the last line we use a tilde followed by an equal sign: `~=`.)
```
Comment (ekohl) [inline en/chromebook_setup/instructions.md]: Should we drop the parenthesis here?

File: en/deploy/README.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ $ git commit -m "My Django Girls app, first commit"
 
 ## Pushing your code to GitHub
 
-Go to [GitHub.com](https://www.github.com) and sign up for a new, free user account. (If you already did that in the workshop prep, that is great!) Be sure to remember your password (add it to your password manager, if you use one).
+Go to [GitHub.com](https://www.github.com) and sign up for a new, free user account. (If you already did that in the workshop prep, that is great!) Be sure to remember your password. (Add it to your password manager, if you use one.)
```
Comment (ekohl) [inline en/deploy/README.md]: Should we drop the parenthesis?

File: en/deploy/signup_pythonanywhere.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 PythonAnywhere is a service for running Python code on servers "in the cloud". We'll use
 it for hosting our site, live and on the Internet.
 
-We will be hosting the blog we're building on PythonAnywhere. Sign up for a "Beginner" account on PythonAnywhere (the free tier is fine, you don't need a credit card).
+We will be hosting the blog we're building on PythonAnywhere. Sign up for a "Beginner" account on PythonAnywhere. (The free tier is fine, you don't need a credit card.)
```
Comment (ekohl) [inline en/deploy/signup_pythonanywhere.md]: Another case where I think the parenthesis could also be dropped.

File: en/deploy/signup_pythonanywhere.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 
 * [www.pythonanywhere.com](https://www.pythonanywhere.com/)
 
 ![The PythonAnywhere signup page showing button to create a the free 'Beginner' account](../deploy/images/pythonanywhere_beginner_account_button.png)
 
-> **Note** When choosing your username here, bear in mind that your blog's URL will take the form `yourusername.pythonanywhere.com`, so choose either your own nickname or a name for what your blog is all about. Also, be sure to remember your password (add it to your password manager, if you use one).
+> **Note** When choosing your username here, bear in mind that your blog's URL will take the form `yourusername.pythonanywhere.com`, so choose either your own nickname or a name for what your blog is all about. Also, be sure to remember your password. (Add it to your password manager, if you use one.)
```
Comment (ekohl) [inline en/deploy/signup_pythonanywhere.md]: And here

File: en/django_forms/README.md [hunk 6]
```diff
@@ -196,7 +196,7 @@ It's time to fill in the dots `[...]`. If `method` is `POST` then we want to con
 form = PostForm(request.POST)
 ```
 
-The next thing is to check if the form is correct (all required fields are set and no incorrect values have been submitted). We do that with `form.is_valid()`.
+The next thing is to check if the form is correct. (All required fields are set and no incorrect values have been submitted.) We do that with `form.is_valid()`.
```
Comment (ekohl) [inline en/django_forms/README.md]: IMHO the previous version was clearer. It's an explanation of correct and this decouples it.

File: en/django_forms/README.md [hunk 7]
```diff
@@ -420,7 +420,7 @@ $ git pull
 [...]
 ```
 
-(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets).
+(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets.)
```
Comment (ekohl) [inline en/django_forms/README.md]: Also questioning the parenthesis here

File: en/dynamic_data_in_templates/README.md [hunk 8]
```diff
@@ -56,7 +56,7 @@ The last missing part is passing the `posts` QuerySet to the template context. D
 
 Please note that we create a *variable* for our QuerySet: `posts`. Treat this as the name of our QuerySet. From now on we can refer to it by this name.
 
-In the `render` function we have one parameter `request` (everything we receive from the user via the Internet) and another giving the template file (`'blog/post_list.html'`). The last parameter, `{}`, is a place in which we can add some things for the template to use. We need to give them names (we will stick to `'posts'` right now). :)  It should look like this: `{'posts': posts}`. Please note that the part before `:` is a string; you need to wrap it with quotes: `''`.
+In the `render` function we have one parameter `request` (everything we receive from the user via the Internet) and another giving the template file (`'blog/post_list.html'`). The last parameter, `{}`, is a place in which we can add some things for the template to use. We need to give them names. (We will stick to `'posts'` right now.) :)  It should look like this: `{'posts': posts}`. Please note that the part before `:` is a string; you need to wrap it with quotes: `''`.
```
Comment (ekohl) [inline en/dynamic_data_in_templates/README.md]: Here you also decouple the explanation of names so IMHO it's less clear what it refers to.

File: en/chromebook_setup/instructions.md (hunk 2 above)
Comment (jhodgdon-drp) [reply en/chromebook_setup/instructions.md]: I agree with dropping parens here.

File: en/deploy/README.md (hunk 1 above)
Comment (jhodgdon-drp) [reply en/deploy/README.md]: I like ekohl's suggestion here.

File: en/deploy/README.md (hunk 3 above)
Comment (jhodgdon-drp) [reply en/deploy/README.md]: How about for the second sentence:

Be sure to remember your password -- add it to your password manager, if you use one.

File: en/deploy/signup_pythonanywhere.md (hunk 4 above)
Comment (jhodgdon-drp) [reply en/deploy/signup_pythonanywhere.md]: I agree with dropping the parens.

File: en/deploy/signup_pythonanywhere.md (hunk 5 above)
Comment (jhodgdon-drp) [reply en/deploy/signup_pythonanywhere.md]: Suggest as above:

Also, be sure to remember your password -- add it to your password manager, if you use one.

File: en/django_forms/README.md (hunk 6 above)
Comment (jhodgdon-drp) [reply en/django_forms/README.md]: I agree with ekohl here.

File: en/django_forms/README.md (hunk 7 above)
Comment (jhodgdon-drp) [reply en/django_forms/README.md]: Agreed that the parens should be removed.

File: en/dynamic_data_in_templates/README.md (hunk 8 above)
Comment (jhodgdon-drp) [reply en/dynamic_data_in_templates/README.md]: Agreed with @ekohl -- the original version is more clear.

File: en/extend_your_application/README.md [hunk 9]
```diff
@@ -186,7 +186,7 @@ $ git pull
 [...]
 ```
 
-(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets).
+(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets.)
```
Comment (jhodgdon-drp) [inline en/extend_your_application/README.md]: I think we should remove the parens here too.

File: en/html/README.md [hunk 10]
```diff
@@ -206,9 +206,9 @@ $ git pull
 [...]
 ```
 
-(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets).
+(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets.)
```
Comment (jhodgdon-drp) [inline en/html/README.md]: I think we should remove the parens here too.

File: en/installation/README.md [hunk 11]
```diff
@@ -34,7 +34,7 @@ data-id="chromebook_setup" data-collapse=true ces-->
 {% include "/deploy/install_git.md" %}
 
 # Create a GitHub account
-Go to [GitHub.com](https://www.github.com) and sign up for a new, free user account. Be sure to remember your password (add it to your password manager, if you use one).
+Go to [GitHub.com](https://www.github.com) and sign up for a new, free user account. Be sure to remember your password. (Add it to your password manager, if you use one.)
```
Comment (jhodgdon-drp) [inline en/installation/README.md]: Suggest, as above:
Be sure to remember your password -- add it to your password manager, if you use one.

**Comment by das-g** [comment]
Let's close this PR for now. I might revisit this topic at a later point in time. Feel free to otherwise improve the pose meanwhile. (Better rephrasing, removal of parentheses, etc.)

**Pull request closed without merging**